src/uct/sm/scopy/cma/cma_iface.c

```diff
@@ -68,6 +71,16 @@ static ucs_status_t uct_cma_iface_query(uct_iface_h tl_iface,
     return UCS_OK;
 }
 
+static inline int uct_cma_has_vm_perms(pid_t UCS_V_UNUSED peer_pid)
+{
+#ifdef HAVE_LINUX_KCMP_H
+    return (syscall(SYS_kcmp, getpid(), peer_pid, KCMP_VM, 0, 0) >= 0) ||
+        (errno != EPERM);
+#else
+    return 1; /* Try anyways */
+#endif
+}
```

Comment:
maybe we can try process_vm_readv of some dummy address?
I would assume that if the process is accessible we would wither be successful or get EFAULT. and if it's not accessible it would be EPERM.
This way would not depend on new kernel to check the readability

Reply:
fixed